src/pages/students/Students.tsx

```diff
@@ -1,11 +1,192 @@
-import './Students.css'
+import "./Students.css";
+import { useForm } from "react-hook-form";
+import type { Resolver } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { studentsScheme } from "../students/students.scheme";
+import { MOCKED_PAYMENT_TYPES } from "../../mocks/paymentTypes";
+import { useNavigate } from "react-router-dom";
```

Comment:
Aca en vez de importarlo de `react-router-dom` lo haces de `react-router`

```suggestion
import { useNavigate } from "react-router";
```